cmd/noms/diff/diff_test.go

```diff
@@ -129,6 +141,10 @@ func TestNomsSetDiff(t *testing.T) {
 -   }
   }
 `
+	expectedPaths2 := []string{
+		`[#3rsij03n33f3qefnfcf9kmh9qubst0km]`,
```

Comment:
Could you construct these strings instead of hard coding them? It's a weaker assertion otherwise.